app/views/admin/admin_users/_user_history.html.erb

```diff
@@ -13,17 +13,39 @@
       <% unless @log_items.blank?
         @log_items.each do |item| %>
         <tr>
-          <td><%= item.created_at %></td>
+          <td><%= item.created_at %> ( <abbr class="timeago" title="<%= item.created_at %>"><%= item.created_at.iso8601  %></abbr> )</td>
           <td><%= log_item_action_name(item.action) %><%= item.role.name if item.role %><%= item.enddate if item.enddate %></td>
           <td><%= item.note %></td>
         </tr>
         <% end %>
       <% end %>
       <tr>
-        <td><%= @user.created_at %></td>
+        <td><%= @user.created_at %> ( <abbr class="timeago" title="<%= @user.created_at %>"><%= @user.created_at.iso8601  %></abbr> )</td>
```

Comment:
There shouldn't be a space between the parentheses as the `<abbr>` tags. It should be `(<abbr class="timeago" title="<%= @user.created_at %>"><%= @user.created_at.iso8601 %></abbr>)`. There's also an extra space in `<%= @user.created_at.iso8601  %>` that should be removed.